public/js/tracking_protection.js

```diff
@@ -1,38 +1,10 @@
 /* eslint-env browser */
 "use strict";
-
-/* 
-	Respect user privacy settings and honor user privacy choices.
-	From Schalk Neethling at https://github.com/schalkneethling/dnt-helper */
-	
-const trackingProtection = {};
-trackingProtection.enabled = function(dnt, ua){
-	window.dataLayer = window.dataLayer || [];
-	let userPrivacySettings = navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack;
-	const userAgent = ua || navigator.userAgent;
-	const anomalousWinVersions = ["Windows NT 6.1", "Windows NT 6.2", "Windows NT 6.3"];
-	const fxMatch = userAgent.match(/Firefox\/(\d+)/);
-	const ieRegEx = /MSIE|Trident/i;
-	const isIE = ieRegEx.test(userAgent);
-	const platform = userAgent.match(/Windows.+?(?=;)/g);
-	if(isIE&&typeof Array.prototype.indexOf !== "function") {
-		return false;
-	} 
-	else if(fxMatch && parseInt(fxMatch[1],10)<32) {
-		userPrivacySettings="Unspecified";
-	} 
-	else if(isIE && platform && anomalousWinVersions.indexOf(platform.toString()) !== -1) {
-		userPrivacySettings="Unspecified";
-	}
-	else {
-		userPrivacySettings = { 0 : "Disabled", 1 : "Enabled" }[userPrivacySettings] || "Unspecified";
-	}
-	return userPrivacySettings === "Enabled"?true:false;
-};
-
 /* global ga */
+/* global _dntEnabled  */
 
-if(trackingProtection && !trackingProtection.enabled()) {
+const trackingProtection = _dntEnabled();
+if(!trackingProtection) {
```

Comment:
not sure if you want to create the intermediate `trackingProtection` const, versus just doing:

```js
if (!_dntEnabled()) {...}
```

But that's just a ridiculous nit, and def not worth holding any merge action happenin'.

Reply:
hahahaha no this is better.  I was console logging the const in various browsers/states of tracking-protection-enablement just to be extra sure but should have deleted it. will fix in next push! no idea on circle but checking it out now.